scripts/conda/meta.yaml

```diff
@@ -0,0 +1,61 @@
+# build a conda package from a local git checkout
+package:
+  name: coremltools
+  version: {{ GIT_DESCRIBE_TAG }}
+
+source:
+  path: ../..  
+
+build:
+  number: {{ GIT_DESCRIBE_NUMBER|int }}
+  string: py{{ CONDA_PY }}_{{GIT_DESCRIBE_HASH}}_{{ GIT_DESCRIBE_NUMBER }}
+
+requirements:
+  build:
+    - cmake
+    - make
+    #- {{ compiler('c') }}
+    #- {{ compiler('cxx') }}
+  host:
+    - python
+    - numpy
+    - pip
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - protobuf >=3.1.0
+    - six >=1.1.0
+    - attrs
+    - sympy
+    - scipy
+    - tqdm
+    - packaging
+  run_constrained:
+    - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx]
+    - pytorch >=1.4,<1.7  # inferred by testing
+
+test:
+  imports:
+    - coremltools
+  commands:
+    # TODO: change to running `coremlconverter --help`
```

Comment:
What is the upstream bug?

Reply:
Good question.  This came over from the conda-forge recipe that Tom wrote.  I will investigate and figure out if this is still necessary.

Reply:
not sure what this was, but the problem is gone, so I've updated the script to call `coremlconverter --help` in the test.